config-bootstrapper/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ tasks.register<com.bmuschko.gradle.docker.tasks.network.DockerRemoveNetwork>("re
 }
 
 tasks.register<DockerPullImage>("pullMongoImage") {
-  image.set("mongo:4.2.6")
+  image.set("mongo:4.4.0")
```

Comment:
curious about this change, is this absolutely necessary? not saying we shouldn't I'd just say it is better to update one thing at the time.

Reply:
Even for integration tests?

Reply:
I would say more so for tests, we should try to change the tests as less as possible (unless absolutely necessary). Again not saying we should not, I was more curious if this upgrade was making it possible to achieve the goal of this PR.

Reply:
I was trying to bring it up to the same version we are using in other integration tests. That being said, it's completely independent of the goal of the PR